<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="mainTab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Network</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="LayoutNetwork"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>DataBase</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTabWidget" name="PlateTab">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="Plate">
               <attribute name="title">
                <string>Plate / User</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="1" column="0" colspan="5">
                   <widget class="QTableView" name="tableView">
                    <property name="sortingEnabled">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="3">
                   <widget class="QPushButton" name="pushButton_7">
                    <property name="text">
                     <string>Add Row</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="3">
                   <widget class="QPushButton" name="pushButton_13">
                    <property name="text">
                     <string>Remove Rows</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3" rowspan="2">
                   <widget class="QPushButton" name="pushButton_9">
                    <property name="text">
                     <string>Submit</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="4" rowspan="2">
                   <widget class="QPushButton" name="pushButton_10">
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="5">
                   <widget class="QLineEdit" name="lineEdit_2"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="Users">
               <attribute name="title">
                <string>Image</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="2" column="0" colspan="3">
                   <widget class="QPushButton" name="pushButton_6">
                    <property name="text">
                     <string>Add Row</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="5">
                   <widget class="QTableView" name="tableView_2">
                    <property name="sortingEnabled">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="3">
                   <widget class="QPushButton" name="pushButton_12">
                    <property name="text">
                     <string>Delete Rows</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3" rowspan="2">
                   <widget class="QPushButton" name="pushButton_8">
                    <property name="text">
                     <string>Submit</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="4" rowspan="2">
                   <widget class="QPushButton" name="pushButton_11">
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="5">
                   <widget class="QLineEdit" name="lineEdit_3"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="Log">
               <attribute name="title">
                <string>Log</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="1" column="0" colspan="4">
                   <widget class="QTableView" name="tableView_3">
                    <property name="sortingEnabled">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3">
                   <widget class="QPushButton" name="pushButton_15">
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QPushButton" name="pushButton_16">
                    <property name="text">
                     <string>Submit</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QPushButton" name="pushButton_14">
                    <property name="text">
                     <string>Delete Rows</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="4">
                   <widget class="QLineEdit" name="lineEdit_4"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="ButtonUpdateDB">
              <property name="text">
               <string>Update Database</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
